strider/trapi.py

```diff
@@ -58,11 +58,10 @@ def get_curies(message: Message) -> list[str]:
     """
     curies = set()
     if 'query_graph' in message:
-        curies |= {
-            qnode.get("id")
-            for qnode in message['query_graph']['nodes'].values()
-            if qnode.get("id", False)
-        }
+        for qnode in message['query_graph']['nodes'].values():
+            qnode_id = qnode.get("id", False)
+            if qnode_id:
```

Comment:
This is a good example use case for the walrus operator.

Reply:
How should this use the walrus operator? Do you mean something like this:
```python
qnode_id |= qnode.get("id", None)
```

Reply:
`if qnode_id := qnode.get("id", False):` I think.
https://www.python.org/dev/peps/pep-0572/